lib/services/UserService

```diff
@@ -0,0 +1,30 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const User = require('../models/User');
+
+module.exports = class UserService {
+  static async create({  email, firstName, lastName, password }) {
+    const passwordHash = await bcrypt.hash(
+      password, Number(process.env.SALT_ROUNDS)
+    );
+    const user = await User.insert({
+      email, firstName, lastName, passwordHash
+    }); return user;
+  }
+
+  static async signIn({ email, password = '' }) {
+    try {
+      const user = await User.getByEmail(email);
```

Comment:
Consider that we could use this getByEmail function in our User Service to check if a new user's email exists, and send an error back if so!